README.md

```diff
@@ -60,6 +60,41 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 Done!
 
+### Semi Dockerized Setup
+
+[osx catalina, reinstall command line tools](https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d)
+
+1.  Pre reqs (macOs instructions)
+
+        brew install postgresql llvm openssl yarn
+
+        echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile
+        export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
+
+2.  Install dependencies
+
+        poetry install
+
+        yarn install
+
+3.  env values
+
+        .env:
```

Comment:
It looks like this needs to be set at the root, not in /app

Reply:
Yep exactly, would be helpful to specify which folders things take place in 👍